docs/install-overview.md

```diff
@@ -6,6 +6,16 @@ In version 0.7.0 and later, you can run Ark in any namespace, which requires add
 
 In version 0.9.0 and later, you can use Ark's integration with restic, which requires additional setup. See [restic instructions][20].
 
+## Customize configuration
+
+Whether you run Ark on a cloud provider or on-premises, you can specify the name of the default volume snapshot when you set up the server by running this additional command:
+
+```bash
+ark server --default-volume-snapshot-locations <location-1>,<location-2>,<location-3>,...
```

Comment:
This isn't a command the user would run, they would amend their Ark deployment YAML to include this flag.

Also, the value of the flag should be formatted as `provider-1:location-1,provider-2,location-2,...`

Reply:
lol this is what I originally thought. Overthinkers r us.